site/cypress/integration/ui_tests/e2e_tests/08-draft-article.js

```diff
@@ -4,10 +4,13 @@
 const article = require('../../../fixtures/fragments/article');
 
 describe("Draft articles", function() {
-    it('An article with the \'draft\' tag should not be visible [ONS-232]', function () {
+    it('An article with the \'draft\' tag should not be visible [ONS-232] and [ONS-351]', function () {
```

Comment:
could update the text to say 'draft tag or draft-reason' (which isn' a tag)

Reply:
thx & done